examples/independent_tasks/independent_tasks.cc

```diff
@@ -0,0 +1,53 @@
+#include <celerity.h>
```

Comment:
Any background on why this is a new example? What user-facing feature of Celerity does it demonstrate? To me it looks like more of a `runtime_test`.

Reply:
It's an example because I needed the ability to easily invoke it standalone with different arguments for the paper extension I'm writing ;)

Which is obviously not a good reason, I'll change that to something else.

Reply:
This is now a new benchmark (of a new type, since we didn't really have full system benchmarks so far, so in a new file).